pkg_ci.Rmd

```diff
@@ -6,9 +6,20 @@ This chapter summarizes our guidelines about continuous integration after explai
 Along with [last chapter](#building), it forms our guidelines for Software Peer Review.
 ```
 
+## What is continuous integration (CI)?
+
+Continuous integration automatically runs tests on software.
+In the case of rOpenSci, CI practically means that a set of tests will be automatically run via github, every time that you make a commit or pull request to github.
+
+For CI to be meaningful you have to have tests set up that work locally using `R CMD check`, see [testing](/building.html#testing).
+It is possible to set up CI before your tests are written, then CI will run the tests as you commit them to the repository.
+
 ## Why use continuous integration (CI)?
 
-All rOpenSci packages must use one form of continuous integration. This ensures that all commits, pull requests and new branches are run through `R CMD check`. rOpenSci packages' continuous integration must also be linked to a code coverage service, indicating how many lines are covered by unit tests.
+All rOpenSci packages must use one form of continuous integration.
+This ensures that all commits, pull requests and new branches are run through `R CMD check`.
+Tests are required to pass before merging pull requests, which provides another layer of information about problems and protection against breaking your package.
```

Comment:
This is not technically a requirement, so I'd prefer that statement to be modified or removed. Temporary breakages can arise through external reasons, so we can not and do not have an absolute requirement of passing tests at all times. The requirement amounts to something more like, "The latest avialable measure of test coverage should be at least 75%." Of course we'd never phrase it that way, so just declare a minimal coverage requirement and presume it to be current, whatever that may actually mean in practice.